packages/ui-list/src/InlineList/index.tsx

```diff
@@ -44,8 +45,7 @@ class InlineList extends Component<InlineListProps> {
   static defaultProps = {
     children: null,
     itemSpacing: 'none',
-    // @ts-expect-error ts-migrate(6133) FIXME: 'el' is declared but its value is never read.
-    elementRef: (el) => {},
+    elementRef: () => {},
```

Comment:
This can be deleted from defaultProps, there is a check for it on line 64. The `children: null` can be deleted too.